src/SourceBuild/content/repo-projects/format.proj

```diff
@@ -2,9 +2,17 @@
 
   <ItemGroup>
     <RepositoryReference Include="arcade" />
+    <RepositoryReference Include="command-line-api" />
     <RepositoryReference Include="roslyn" />
+    <RepositoryReference Include="runtime" />
+    <RepositoryReference Include="symreader" />
```

Comment:
Is symreader really a live dependency for the MSFT build?

Reply:
Fixed in https://github.com/dotnet/installer/pull/18557/commits/5f14aa31ff1c07afde45b82fc5f5e9e9848231d5